sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/plans/logical/sql/ParsedStatementSuite.scala

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql.catalyst.plans.logical.sql
+
+import org.apache.spark.SparkFunSuite
+import org.apache.spark.sql.catalyst.expressions.Attribute
+import org.apache.spark.sql.catalyst.plans.logical.LogicalPlan
+
+class ParsedStatementSuite extends SparkFunSuite {
+
+  test("Map[String, String] field") {
+    case class TestStatement(p: Map[String, String]) extends ParsedStatement {
+      override def output: Seq[Attribute] = Nil
+      override def children: Seq[LogicalPlan] = Nil
+    }
+
+    TestStatement(Map("abc" -> "def")).toString
+  }
+
+  test("Map[String, Int] field") {
```

Comment:
If this is a bug, we need JIRA ID here. (For me, this issue looks like improvements.)

Reply:
JIRA SPARK-27947